backend/Testing/ApiTests/ApiTestBase.cs

```diff
@@ -14,6 +14,7 @@ public class ApiTestBase
     public string BaseUrl => TestingEnvironmentVariables.ServerBaseUrl;
     private readonly SocketsHttpHandler _httpClientHandler;
     public readonly HttpClient HttpClient;
+    public string? CurrJwt { get; private set; }
```

Comment:
is this actually used anywhere? I don't see it

Reply:
The LexboxProject class reads.